### ISSUE TITLE
Add task to check if on arm platform, and redirect to geerlingguy.docker_arm

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/ansible-role-docker#290]
I've had a number of issues on this project lately asking about ARM installation issues... since the ARM setup process is different enough, I maintain a separate role (https://github.com/geerlingguy/ansible-role-docker_arm) for ARM devices. I should add a task to this role that checks the current platform, and if it's ARMv7/ARM64, it fails and directs the user to use the docker_arm role instead.

**Comment by j-koehler** [comment]
From my experience since I recently used this role to provision a Raspi (and only knew the "main role" and did not know about a special docker-arm-role 🙈 ): the only difference is to figure out which architecture you have to set for docker and for docker-compose (since they differ, which is more of a problem on Docker side (imho)):

```
docker_compose_arch: armv7
docker_apt_arch: armhf
```

with this, everything went super smooth and I have a nice and cozy with all the features of the "main role" :)

Not sure how hard adding or detecting `arm` defaults to this role would be? 

If this is too hard (e.g. too many different variables to consider) maybe it's enough to just print this as a notification when arm architecture is detected (e.g. "arm architecture detected which is not officially supported by this role - consider switching to geerlingguy.docker_arm") and not fail hardcoded?

**Comment by geerlingguy** [comment]
At this point (about a year later), it seems like the Docker repos support ARM more easily, and it might be okay to put the arm-based role back into this one, if a custom installation isn't needed.

I'd have to do more testing, but that would be desirable as this role has some features the arm one is lacking. Plus it's used more widely.

**Comment by j-koehler** [comment]
> about a year later

Yeah no worries, I just checked the issues with "planned" label assuming this would be "next". :)

**Comment by geerlingguy** [comment]
Heh... "planned" just means (for me, at least) I 'plan' on doing something with the issue, and I don't want it to auto-close :)

**Comment by geerlingguy** [comment]
See: https://github.com/geerlingguy/ansible-role-docker/pull/376

**Comment by geerlingguy** [comment]
Deprecation issue for the docker_arm role: https://github.com/geerlingguy/ansible-role-docker_arm/issues/38
